src/store/gameStore.ts

```
@@ -0,0 +1,70 @@
+import { create } from 'zustand';
+import type { CellValue, Grid, Tetromino, GameState as IGameState } from '../types/game';
```

ESLint

[error] 'Tetromino' is defined but never used.

(@typescript-eslint/no-unused-vars)